package.json

```diff
@@ -12,8 +12,7 @@
     "component library"
   ],
   "type": "module",
-  "main": "./lib/index.cjs",
-  "module": "./lib/index.js",
```

Comment:
Lots of info is pointing to this key being a problem, but there is some documentation about it here:

https://esbuild.github.io/api/#main-fields